backend/app/Http/Controllers/UserController.php

```diff
@@ -77,4 +77,28 @@ public function visitUser($user_id)
 			?response()->json(['data' => $user])
 			:response()->json(['error' => 'User not found'], 404);
 	}
+
+	public function update(Request $request, $user_id)
+	{
+		try {
+			$validatedData = $request->validate([
+				'name' => 'required|string',
+				'email' => 'required|string|email|unique:users',
+				'password' => 'required|string',
+				'profile_picture' => 'string'
+			]);
+		
+			$user = User::findOrFail($user_id);
+			$user->name = $validatedData['name'];
+			$user->email = $validatedData['email'];
+			$user->password = Hash::make($validatedData['password']);
```

Comment:
Exclude email, password and profile picture, should be different api.
You dont need to update email, password, profile everytime you change something in your personal details.

Reply:
Okay sir I will separate nya sa profile picture nya remove email and password. Nya ako ibilin ang name details nalang nya sir :bow: